applications/machine_learning/configuration/nrf5340dk_nrf5340_cpuapp/child_image/mcuboot/prj.conf

```diff
@@ -64,3 +64,6 @@ CONFIG_FLASH_SIMULATOR_STATS=n
 # Enable custom command to erase settings partition.
 CONFIG_ENABLE_MGMT_PERUSER=y
 CONFIG_BOOT_MGMT_CUSTOM_STORAGE_ERASE=y
+
+# Use minimal C library instead of the Picolib
```

Comment:
btw. Should also we update also MCUboot used by Thingy:53 (needs change in the sdk-mcuboot) or we allow it to switch to picolibc?

Reply:
IMHO everything MCUboot should be set to MINI, but I have seen that some of provided commits skip things like 7002.